library/spark/spark-accumulo-library/src/main/java/uk/gov/gchq/gaffer/sparkaccumulo/operation/handler/dataframe/FiltersToOperationConverter.java

```diff
@@ -215,7 +214,7 @@ private View applyGroupFilters(final View view) {
         final Map<String, List<ConsumerFunctionContext<String, FilterFunction>>> groupToFunctions = new HashMap<>();
         for (final Filter filter : filters) {
             final Map<String, List<ConsumerFunctionContext<String, FilterFunction>>> map = getFunctionsFromFilter(filter);
-            for (final Map.Entry<String, List<ConsumerFunctionContext<String, FilterFunction>>> entry : map.entrySet()) {
+            for (final Entry<String, List<ConsumerFunctionContext<String, FilterFunction>>> entry : map.entrySet()) {
```

Comment:
Are we going to move away from fully qualified names for inner classes? I've noticed that fully qualified references are used throughout the project, we should probably be consistent and detail this in the Wiki.